class.jetpack.php

```diff
@@ -2834,6 +2837,22 @@ public static function get_additional_stat_data( $prefix = '' ) {
 		return $return;
 	}
 
+	private static function get_site_user_count() {
+		global $wpdb;
+
+		if ( function_exists( 'wp_is_large_network' ) ) {
+			if ( wp_is_large_network( 'users' ) ) {
+				return 10101; // Not a real value but should tell us that we are dealing with a large network.
```

Comment:
The problem with using get_user_count is that it would give us the number of users in the network for each of the sites on the network. Which is not really true for the true.